src/api/config/routes/webui_routes.rb

```diff
@@ -392,4 +392,6 @@
       end
     end
   end
+
+  resources :workflow_runs, only: [:index], controller: 'webui/workflow_runs'
```

Comment:
Would it be a good idea to have a nested route, something like `token_id/workflow_runs`?

Reply:
yes, I moved it under the tokens resources :)